### ISSUE TITLE
Distance() function depends on parameter order

### ISSUE THREAD

**Issue by StefanFaltz** [EDCD/EDDI#1911]
## EDDI version in which issue is found
3.7.0

## VoiceAttack version in which issue is found (as applicable)
It's possible to reproduce with VoiceAttack (v1.8.6) running and EDDI standalone.

## Steps to reproduce

I visited Sol before so this is in the EDDI database.
Ship is docked in Kamadhenu.

Debug script:

```
Current system is {system.name}.
{Pause(1000)}
Distance from {system.name} to Sol is {Distance(system.name, "Sol")}.
{Pause(1000)}
Distance from Sol to {system.name} is {Distance("Sol", system.name)}.
{Pause(2000)}
```

## Expected
Calculated distance should match irrespective of parameter order to Distance() function.

## Observed
Current system is Kamadhenu.
Distance from Kamadhenu to Sol is 149.24.
Distance from Sol to Kamadhenu is 110.

## Investigation
n/a

**Comment by StefanFaltz** [comment]
Didn't see it before but might be related to #1884

**Comment by Tkael** [comment]
Likely already fixed by #1884. :-)

**Comment by Tkael** [comment]
Closing as already resolved for the next release but thanks for the ticket. o7
